### PR TITLE
1176 remove `Show Warnings` from modules

### PR DESCRIPTION
**Pull request by m7pr**
Part of https://github.com/insightsengineering/teal/issues/1176

### CODE REVIEW

**Comment by m7pr** [comment]
I have read the CLA Document and I hereby sign the CLA

**Review by kartikeyakirar — APPROVED** [review]
LGTM!! no more occurrence of `teal.code::get_warnings()`. Could you please add a news entry for this?

File: R/tm_a_pca.R [hunk 1]
```diff
@@ -282,7 +282,6 @@ ui_a_pca <- function(id, ...) {
         )
       ),
       forms = tagList(
```
Comment (kartikeyakirar) [inline R/tm_a_pca.R:284]: Do we need `taglist()` now ?

Comment (m7pr) [reply R/tm_a_pca.R:284]: replied the same as in the tmc, I think this is fine and shows the ability for this part to be extendable

**Pull request merged**